azure-pipelines/nightly-snapshot-releases/Nightly Common VSTS Release.yml

```diff
@@ -1,108 +1,34 @@
-# 'Allow scripts to access the OAuth token' was selected in pipeline.  Add the following YAML to any steps requiring access:
-#       env:
-#           MY_ACCESS_TOKEN: $(System.AccessToken)
-# Variable 'android:serverUrl' was defined in the Variables tab
-# Variable 'ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME' was defined in the Variables tab
-# Variable 'test_repo_branch' was defined in the Variables tab
-# Variable 'test_repo_dir' was defined in the Variables tab
-# Variable Group 'Android Common Variables' was defined in the Variables tab
-# Cron Schedules have been converted using UTC Time Zone and may need to be updated for your location
+# File: azure-pipelines\nightly-snapshot-releases\Nightly Common VSTS Release.yml
+# Description: Assemble & publish Snapshot common release nightly to internal feed
+# https://identitydivision.visualstudio.com/Engineering/_packaging?_a=feed&feed=AndroidADAL
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger: none
+pr: none
+
 resources:
-  pipelines:
-  - pipeline: Nightly ADAL VSTS Release
-    source: Nightly ADAL VSTS Release
-    trigger:
-      branches:
-        include:
-        - dev
+# TODO uncomment once we have a valid artifact in the pipeline
+#  pipelines:
+#  - pipeline: Android\Nightly Snapshot Releases\Nightly ADAL VSTS Release
+#    project: Engineering
+#    source: Nightly ADAL VSTS Release
+#    trigger:
+#      branches:
+#        include:
+#        - dev
```

Comment:
For better tracking, I would recomment to rather open a Ado workitem to add it once we have valid artifact, instead of adding comments and TODO here

Reply:
https://identitydivision.visualstudio.com/Engineering/_workitems/edit/1384923